kpi/utils/strings.py

```diff
@@ -10,3 +10,8 @@ def to_str(obj):
     if isinstance(obj, bytes):
         return obj.decode()
     return obj
+
+
+def split_lines_to_list(value: str) -> list:
```

Comment:
nit: i would use `values` instead of `ip_addresses` since this could be used for any strings. not blocking

Reply:
💯